docker-compose.yml

```diff
@@ -70,6 +70,7 @@ services:
             - ./client/src:/var/www/src:ro
             - ./client/tests:/var/www/tests
             - ./client/phpstan.neon:/var/www/phpstan.neon
+            - ./client/web:/var/www/web
```

Comment:
This needs to be re-removed. It causes problems with a fresh build because you generate the assets in `docker-compose build` and then overwrite them with whatever's in your local folder in `docker-compose up`. Normally this results in there being two assets folders and the application getting confused.

Certainly something to fix in the future (perhaps after the folder restructure in v4).

Reply:
Ahhh OK - was just getting annoyed by having to sync the folders up after a yarn build.

Reply:
Yeah, it's a massive PITA when you're doing CSS/JS work.